tests/functional/api/test_query.py

```diff
@@ -7,3 +11,28 @@ def test_basic_query(eth_tester_provider):
     df = chain.blocks.query("number", "timestamp")
     assert len(df) == 4
     assert df["timestamp"][3] > df["timestamp"][2] >= df["timestamp"][1] >= df["timestamp"][0]
+    df_all = chain.blocks.query("*")
+    columns = list(df_all.columns)
+    assert [
+        "gas_data",
+        "consensus_data",
+        "hash",
+        "number",
+        "parent_hash",
+        "size",
+        "timestamp",
+    ] == columns
```

Comment:
Little easier to read:
```suggestion
    assert columns == [
        "gas_data",
        "consensus_data",
        "hash",
        "number",
        "parent_hash",
        "size",
        "timestamp",
    ]
```